Junk4Dollars-app/Info.plist

```diff
@@ -35,6 +35,8 @@
     <true/>
     <key>NSAppTransportSecurity</key>
     <dict>
+        <key>NSAllowsArbitraryLoads</key>
```

Comment:
What was this needed for?

Reply:
It was complaining about using non `https` endpoints